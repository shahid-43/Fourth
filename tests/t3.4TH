1 2 3 ^ + . CR .
2 3 4 + * . CR . 