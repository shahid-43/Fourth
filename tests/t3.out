9
14

Execution finished with an empty stack.